datamodel_code_generator/model/base.py

```diff
@@ -64,6 +65,8 @@ def type_hint(self) -> str:
 
         if not type_hint:
             return OPTIONAL
+        elif self.required is False:
+            return f'{UNION}[{type_hint}, Unset]'
```

Comment:
@goodoldneon 
Should we use a built-in special `unset` class in Python or Pydantic?
I'm sorry that I forget the name of the special `unset` class :(
Do you know the name?

Reply:
If there's a builtin class we should definitely use it! But I'm not able to find one 🙁

Reply:
Maybe python's ellipsis would be good fit for this? https://stackoverflow.com/a/47189816/8896457

But it is used already by pydantic, I'm not sure if things won't collide.

Reply:
Please take a look also at this discussion: 
https://chat.stackoverflow.com//transcript/message/54521781#54521781

Reply:
And also this: https://peps.python.org/pep-0661/

Reply:
Pydantic has its own `Undefined` type:

https://github.com/samuelcolvin/pydantic/blob/cc54acb612cb5144a34caebb9ac143324a0cb4a1/pydantic/fields.py#L81

It is now used also in FastAPI (today's update):

https://github.com/tiangolo/fastapi/blob/1876ebc77949a9a254909ec61ea0c09365169ec2/fastapi/dependencies/utils.py#L46